packages/react-components/src/components/MeetingConferencePhoneInfo.tsx

```diff
@@ -0,0 +1,215 @@
+                </Stack.Item>
+                <Stack.Item className={phoneInfoStep}>
+                  {conferencePhoneInfoList.map((phoneNumber, index) => (
+                    <Stack.Item key={index}>
+                      <Text className={phoneInfoTextStyle}>{formatPhoneNumberInfo(phoneNumber, strings)}</Text>
+                    </Stack.Item>
+                  ))}
+                </Stack.Item>
+              </Stack>
+              <Stack
+                horizontal
+                horizontalAlign="space-between"
+                verticalAlign="center"
+                className={phoneInfoInctructionLine}
+              >
+                {}
+                <Stack.Item>
+                  <Stack horizontal>
+                    <Stack.Item className={phoneInfoIcon(theme)}>
+                      <Icon
+                        iconName="DtmfDialpadButton"
+                        style={{ color: theme.palette.themePrimary, padding: '8px' }}
+                      />
+                    </Stack.Item>
+                    <Stack.Item>
+                      <Text className={phoneInfoTextStyle}>{strings?.meetingConferencePhoneInfoModalMeetingId}</Text>
+                    </Stack.Item>
+                  </Stack>
+                </Stack.Item>
+                <Text className={phoneInfoTextStyle}>{conferencePhoneInfoList[0].conferenceId}#</Text>
+              </Stack>
+              <Stack horizontal horizontalAlign="space-between" verticalAlign="center">
+                <Stack horizontal>
+                  <Stack.Item className={phoneInfoIcon(theme)}>
+                    <Icon iconName="PhoneInfoWait" style={{ color: theme.palette.themePrimary, padding: '8px' }} />
+                  </Stack.Item>
+                  <Stack.Item>
+                    <Text className={phoneInfoTextStyle}>{strings?.meetingConferencePhoneInfoModalWait}</Text>
+                  </Stack.Item>
+                </Stack>
+              </Stack>
+            </Stack>
+          )}
+        </Modal>
+      }
+    </>
+  );
+};
+
+/**
+ * @internal
+ * format phone number
+ */
+export const formatPhoneNumber = (phoneNumber: string): string => {
+  const cleaned = ('' + phoneNumber).replace(/\D/g, '');
+  const match = cleaned.match(/^(\d{1,2})?(\d{3})(\d{3})(\d{4})$/);
+  if (match) {
+    const intlCode = '+' + match[1] + ' ';
+    return [intlCode, '(', match[2], ') ', match[3], '-', match[4]].join('');
+  }
+  return phoneNumber;
+};
+
+/**
+ * @internal
+ * format phone number
+ */
+export const formatPhoneNumberInfo = (
+  phoneNumber: _ConferencePhoneInfo | undefined,
+  strings: MeetingConferencePhoneInfoModalStrings | undefined
+): string => {
+  if (!phoneNumber) {
+    return '';
+  }
+  const toll = phoneNumber.isTollFree
+    ? strings?.meetingConferencePhoneInfoModalTollFree
+    : strings?.meetingConferencePhoneInfoModalToll;
+  const countryAndCity = [phoneNumber.country, phoneNumber.city].filter((x) => x).join(', ');
```

Comment:
Are phoneNumber.country and city localized?

Reply:
No, tsCalling provide only English version of it

Reply:
Thinking more, I don't think we can localize this. We'd had to somehow provide localizations for every city in each language. Maybe bring this up in the feature weekly to get others' opinions but don't think we can do anything here

Reply:
Current solution is provide only English version for geo locations.